src/common/pony/detail/atomics.h

```diff
@@ -8,10 +8,12 @@
 #  error "Unsupported platform"
 #endif
 
-#ifndef __cplusplus
+ #ifndef __cplusplus
 #  include <stdalign.h>
 #endif
 
+#include <platform.h>
```

Comment:
this is needed because dragonfly uses GCC but it doesn't use glibc so there's not <features.h> as referenced below. perhaps there is a better way to do this?